backend/controllers/logController.js

```diff
@@ -55,36 +56,29 @@ logController.queryLog = () => {
   }
 };
 
-logController.formatLog = logList => (
-  logArrayConverter(logList)
-);
+logController.formatLog = (logList) => logArrayConverter(logList);
```

Comment:
Not something we need to change with this PR but to go over in our afternoon call—since this function is just the returned result of another function, we should move all the logArrayConverter work into this function.